citrination_client/views/tests/test_model_template_builder.py

```diff
@@ -84,6 +89,110 @@ def test_workflow():
         data_view_id = data_views_client.create(dv_config, "my view", "my description")
         assert data_view_id == 555
 
+        # Update an ML template
+        data_views_client.update("555", dv_config, "my view", "my description")
+
+
+def test_workflow_non_async():
```

Comment:
This test seems mostly redundant with the test above and also doesn't appear to meaningfully test the sync v.s. async flag? I might be misreading it, though 😞

Reply:
I think this is the flags being reversed again -- it's changed now to be more clear (i.e., `True` means "wait")